### PR TITLE
RelativePosition

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Review by IRacle1 — APPROVED** [review]

**Review by xNexus-ACS — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -490,6 +490,15 @@ public Vector3 Position
             set => ReferenceHub.TryOverridePosition(value, Vector3.zero);
         }
 
+        /// <summary>
+        /// Gets or sets the relative player's position.
```
Comment (Thundermaker300) [inline Exiled.API/Features/Player.cs:494]: Add a remark that the value will be 0,0,0,0 if they are not an FPC role.

**Review by Thundermaker300 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Player.cs:494]: Okay

Comment (louis1706) [reply Exiled.API/Features/Player.cs:494]: done

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -490,6 +490,16 @@ public Vector3 Position
...
+        /// </summary>
+        /// <remarks>value will be default if the player it's not an <see cref="FpcRole"/>.</remarks>
```
Comment (Thundermaker300) [inline Exiled.API/Features/Player.cs:496]:
```suggestion
        /// <remarks>The value will be default if the player's role is not an <see cref="FpcRole"/>.</remarks>
```

**Review by Thundermaker300 — CHANGES_REQUESTED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**